### PR TITLE
minor changes

### PR DESCRIPTION
**Pull request by jgabrielius**
minor improvements

### CODE REVIEW

File: University-advisor-web/Models/UniversityReviewModel.cs [hunk 1]
```diff
@@ -39,7 +43,21 @@ public void SaveReviews()
...
+                DuplicateReviewEventArgs args = new DuplicateReviewEventArgs();
+                args.Message = Messages.reviewAlreadySubmitted;
+                args.Id = UniversityId;
+                DuplicateReview?.Invoke(this, args);
+                return true;
+            }
+            return false;
+        }
+
+        public class DuplicateReviewEventArgs : EventArgs
```
Comment (avizen-j) [inline University-advisor-web/Models/UniversityReviewModel.cs:57]: Is this class intentionally left here?

File: University-advisor-web/Controllers/ReviewController.cs [hunk 2]
```diff
@@ -11,6 +13,8 @@ public class ReviewController : Controller
         private readonly ILogger _logger;
         private readonly IErrorHandler _errorHandler;
 
+        public delegate void Print(int value);
```
Comment (avizen-j) [inline University-advisor-web/Controllers/ReviewController.cs]: Is it used anywhere in the code?

File: University-advisor-web/Models/UniversityReviewModel.cs (hunk 1 above)
Comment (jgabrielius) [reply University-advisor-web/Models/UniversityReviewModel.cs:57]: yes

File: University-advisor-web/Controllers/ReviewController.cs (hunk 2 above)
Comment (jgabrielius) [reply University-advisor-web/Controllers/ReviewController.cs]: fixed it in commit

**Pull request merged**